projects/1.16/assets/mystical-world/mysticalworld/lang/zh_cn.json

```
@@ -1 +1,434 @@
+    "block.mysticalworld.blackened_stone_stairs": "漆黑石楼梯",
+    "block.mysticalworld.blackened_stone_wall": "漆黑石墙",
+    "block.mysticalworld.blackened_stone_wide_post": "漆黑石宽柱",
+    "block.mysticalworld.bone_pile_1": "骨堆",
+    "block.mysticalworld.bone_pile_2": "骨堆",
+    "block.mysticalworld.bone_pile_3": "骨堆",
+    "block.mysticalworld.bone_pile_4": "骨堆",
+    "block.mysticalworld.brown_mushroom": "棕色蘑菇",
+    "block.mysticalworld.brown_mushroom_fence": "棕色蘑菇栅栏",
+    "block.mysticalworld.brown_mushroom_fence_gate": "棕色蘑菇栅栏门",
+    "block.mysticalworld.brown_mushroom_full": "完整的棕色蘑菇",
+    "block.mysticalworld.brown_mushroom_slab": "棕色蘑菇台阶",
+    "block.mysticalworld.brown_mushroom_small_post": "棕色蘑菇窄柱",
+    "block.mysticalworld.brown_mushroom_stairs": "棕色蘑菇楼梯",
+    "block.mysticalworld.brown_mushroom_wall": "棕色蘑菇墙",
+    "block.mysticalworld.brown_mushroom_wide_post": "棕色蘑菇宽柱",
+    "block.mysticalworld.charred_fence": "烧焦木栅栏",
+    "block.mysticalworld.charred_fence_gate": "烧焦木栅栏门",
+    "block.mysticalworld.charred_log": "烧焦原木",
+    "block.mysticalworld.charred_planks": "烧焦木板",
+    "block.mysticalworld.charred_slab": "烧焦木台阶",
+    "block.mysticalworld.charred_small_post": "烧焦木窄柱",
+    "block.mysticalworld.charred_stairs": "烧焦木楼梯",
+    "block.mysticalworld.charred_wall": "烧焦木墙",
+    "block.mysticalworld.charred_wide_post": "烧焦木宽柱",
+    "block.mysticalworld.charred_wood": "烧焦木",
+    "block.mysticalworld.copper_block": "铜块",
+    "block.mysticalworld.copper_ore": "铜矿石",
+    "block.mysticalworld.copper_slab": "铜台阶",
+    "block.mysticalworld.copper_small_post": "铜窄柱",
+    "block.mysticalworld.copper_stairs": "铜楼梯",
+    "block.mysticalworld.copper_wall": "铜墙",
+    "block.mysticalworld.copper_wide_post": "铜宽柱",
+    "block.mysticalworld.gall_apple_crop": "瘿苹果",
+    "block.mysticalworld.granite_quartz_ore": "花岗岩石英矿石",
+    "block.mysticalworld.iron_brick": "铁砖块",
+    "block.mysticalworld.iron_brick_slab": "铁砖台阶",
+    "block.mysticalworld.iron_brick_small_post": "铁砖窄柱",
+    "block.mysticalworld.iron_brick_stairs": "铁砖楼梯",
+    "block.mysticalworld.iron_brick_wall": "铁砖墙",
+    "block.mysticalworld.iron_brick_wide_post": "铁砖宽柱",
+    "block.mysticalworld.lead_block": "铅块",
+    "block.mysticalworld.lead_ore": "铅矿石",
+    "block.mysticalworld.lead_slab": "铅台阶",
+    "block.mysticalworld.lead_small_post": "铅窄柱",
+    "block.mysticalworld.lead_stairs": "铅楼梯",
+    "block.mysticalworld.lead_wall": "铅墙",
+    "block.mysticalworld.lead_wide_post": "铅宽柱",
+    "block.mysticalworld.mud_block": "泥块",
+    "block.mysticalworld.mud_block_fence": "泥栅栏",
+    "block.mysticalworld.mud_block_fence_gate": "泥栅栏门",
+    "block.mysticalworld.mud_block_slab": "泥台阶",
+    "block.mysticalworld.mud_block_small_post": "泥窄柱",
+    "block.mysticalworld.mud_block_stairs": "泥楼梯",
+    "block.mysticalworld.mud_block_wall": "泥墙",
+    "block.mysticalworld.mud_block_wide_post": "泥宽柱",
+    "block.mysticalworld.mud_brick": "泥砖块",

```

```suggestion
    "block.mysticalworld.mud_brick": "泥砖",
```
同上，不过这里貌似是有对应物品的
反正先整一个（）